src/Recipe.tsx

```diff
@@ -0,0 +1,25 @@
+import  React  from "react";
+
+import IngredientList from "./IngredientList";
+import {TRecipe} from "./App"
+
+
+type RecipeProps = { recipe: TRecipe};
+
+const Recipe = ({ recipe }: RecipeProps) => (
+  <>
+    <br />
+        <p>Ingredients</p>
+    <span>{` Title: ${recipe.title} `}</span>
```

Comment:
NON-BLOCKING: You'd probably want to swap these two lines but that's just a styling preference on my part.

```
<span>{` Title: ${recipe.title} `}</span>
<p>Ingredients</p>
```